### ISSUE TITLE
Change/simplify the build process

### ISSUE THREAD

**Issue by RoyEJohnson** [openstax/os-webview#2005]
# Description
osweb is built with an array of confusing gulp scripts that are a little hard to follow. Modern build systems (e.g., webpack) can do this all by themselves with a single configuration file.

# Acceptance criteria
osweb uses webpack for both development and production builds.

# Acceptance testing
Regression only. Watch in particular for icons that no longer show up (though I think I've got them all). Would also be good to check some of the analytics tracking stuff.

**Comment by brittanyearly** [comment]
Verified in QA.

**Comment by brittanyearly** [comment]
Verified in Staging.

**Comment by brittanyearly** [comment]
Verified in Production.
